src/model/user.rs

```diff
@@ -169,14 +169,14 @@ impl CurrentUser {
     /// use serenity::model::permissions::Permissions;
     ///
     /// // assuming the cache has been unlocked
-    /// let url = cache.user.invite_url(Permissions::empty());
+    /// let url = cache.user.invite_url(Permissions::empty()).unwrap();
```

Comment:
Also since these tests now have to be `no_run`d, the CurrentUser Id mutation can be removed from both doctests.